diff --git a/node_modules/@chakra-ui/core/dist/es/RadioGroup/index.js b/node_modules/@chakra-ui/core/dist/es/RadioGroup/index.js
index d1bfdaf..69e68e7 100644
--- a/node_modules/@chakra-ui/core/dist/es/RadioGroup/index.js
+++ b/node_modules/@chakra-ui/core/dist/es/RadioGroup/index.js
@@ -55,7 +55,8 @@ var RadioGroup = forwardRef(function (_ref, ref) {
       mb: spacing
     };
     return jsx(Box, _extends({
-      display: isInline ? "inline-block" : "block"
+      display: isInline ? "inline-block" : "block",
+      key: child.key
     }, !isLastRadio && spacingProps), cloneElement(child, {
       size: child.props.size || size,
       variantColor: child.props.variantColor || variantColor,
